### DISCUSSION TITLE
啥时候升级到23.05呀？

### DISCUSSION THREAD

**Discussion by zzzbaba** [istoreos/istoreos#1562] [Ideas]
23.05.3都出来了，咱们还是22.5
内核最新是6.6
啥时候可以升级呀

**Comment by jjm2473** [comment]
23.05的内核是5.15。

**Reply by zzzbaba** [reply]
明白了，期待厂家加快适配，辛苦了

**Reply by jjm2473** [reply]
内核版本不是厂家适配的，看openwrt官方现在用什么版本。

**Comment by zzzbaba** [comment]
难怪我用6.1 6.6总是奇奇怪怪的问题
比如只有IPV6通道，V4不能上网。比如旁路网关总是不能联网等等
而现在istoreOS没有任何问题。

Liangbin Lian ***@***.***> 于2024年5月11日周六 17:53写道：

**Reply by jjm2473** [reply]
你说的这个估计跟内核版本没太大关系，可能某些翻墙插件有问题而已。

**Comment by zzzbaba** [comment]
试过很多版本OP，catwrt esir ImmoralWrt cooluc，在J4125小主机上都存在奇怪问题
不折腾了，稳定就好。谢谢您的回复。

Liangbin Lian ***@***.***> 于2024年5月11日周六 19:52写道：
